brevbaker-dsl/src/main/kotlin/no/nav/pensjon/brev/template/dsl/expression/Number.kt

```diff
@@ -32,7 +32,8 @@ fun Expression<Int>.toYear(): Expression<Year> =
         UnaryOperation.MapValue(IntToYear)
     )
 
-fun Expression<Double>.format(): Expression<String> = format(formatter = LocalizedFormatter.DoubleFormat)
+fun Expression<Double>.formatTwoDecimals(): Expression<String> = format(formatter = LocalizedFormatter.DoubleFormatTwoDecimal)
```

Comment:
Vi kan jo vurdere om vi ønskjer å halde på to desimalar som standard (og da ikkje endre metodenamnet frå format() for den med to desimalar) eller om malutviklaren må ta stilling til antal desimalar kvar gong.Men synes